MANIFESTO.md

```diff
@@ -32,15 +32,11 @@ principles around which we're all aligned that form the ethos of our community.
 Below is a proposal for our principles, most of which were defined at the
 beginning of the program when we first met.
 
-1. Be optimistic
-2. Actively communicate
-3. Be open to new ideas and disagreements
-4. Be the person in the arena
-5. Keep accountable, maintain consistency
-6. Invest in each other -- give before you take
-7. Be curious -- have an insatiable love for learning
-8. Live in the future, then build what's missing
-9. Lean into the unknown
+1. **Give awesome to get awesome** — invest in each other and the program
```

Comment:
These are amazing, thanks so much for writing this up Arushi!! 🙏 Do you want to add a note around how these values are ranked? Thought your explanation was valuable.